internals/generators/organization/config.yml.hbs

```diff
@@ -147,7 +147,7 @@ client:
           pattern_description:
             en: "only letters, numbers, and @/./+/-/_ characters"
           placeholder:
-            en: "enter first name"
+            en: "first name"
```

Comment:
Not related to this PR feature i.e. placeholder and label are the same.

Reply:
you're right, but they were kind of mixed before so I made them all the same, I removed "enter" to keep it short so it always fit on small screens.